OWML.ModHelper.Input/ModInputHandler.cs

```diff
@@ -0,0 +1,440 @@
+﻿using System;
+using System.Reflection;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using OWML.Common;
+using UnityEngine;
+
+namespace OWML.ModHelper
+{
+    public class ModInputHandler : IModInputHandler
+    {
+        private class InputInterceptor
+        {
+            public static void SingleAxisRemovePre(SingleAxisCommand __instance)
+            {
+                ModInputHandler.Self.UnregisterGamesBinding(__instance);
+            }
+
+            public static void DoubleAxisRemovePre(DoubleAxisCommand __instance)
+            {
+                ModInputHandler.Self.UnregisterGamesBinding(__instance);
+            }
+
+            public static void SingleAxisUpdatePost(SingleAxisCommand __instance, ref float ____value, int ____axisDirection, KeyCode ____gamepadKeyCodePositive, KeyCode ____gamepadKeyCodeNegative, KeyCode ____keyPositive, KeyCode ____keyNegative)
```

Comment:
Use line breaks between each argument to avoid extremely long lines:
```suggestion
            public static void SingleAxisUpdatePost(
                SingleAxisCommand __instance,
                ref float ____value,
                int ____axisDirection,
                KeyCode ____gamepadKeyCodePositive,
                KeyCode ____gamepadKeyCodeNegative,
                KeyCode ____keyPositive,
                KeyCode ____keyNegative
            )
```

Reply:
done